task/buildah/0.3/buildah.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: tekton.dev/v1
 kind: Task
 metadata:
   labels:
-    app.kubernetes.io/version: "0.2.1"
+    app.kubernetes.io/version: 0.3.1
```

Comment:
Should this just be X.Y?

Reply:
+1, some tasks are `X.Y`, some are some `X.Y.1`, but they folder are all `X.Y` format

Reply:
I'm not sure what the impact would be, but deciding if X.Y.Z should be disallowed is out of scope in this PR I guess.